src/bics_bot/cogs/events/on_events.py

```diff
@@ -26,6 +29,24 @@ async def on_member_join(self, member: Member):
 
         await member.send(embed=WelcomeEmbed(member.display_name, server.name))
 
+    @tasks.loop(seconds=5.0)
```

Comment:
Time of iteration incorrect

Reply:
Be aware that we want it to repeat every day at 7am for example. And not every 24h from when the bot is started. I have the necessary parameters for that. It took me a while to make it work. So if you can't figure that out, just tell me.

Good Job so far!